src/parser/ast/mod.rs

```diff
@@ -141,6 +141,22 @@ impl PartialEq for DebugSymRef {
     }
 }
 
+/// Interface of the debug symbol reference to use with the Chiquito language server
+pub trait LanguageServerInterface {
```

Comment:
@alxkzmn I don't think it is custom in Rust to call Interface to a trait. Also, the interface is used by the Language Server, not an interface to the Language Server.